packages/sit-onyx/src/components/OnyxPagination/types.ts

```diff
@@ -18,4 +18,11 @@ export type OnyxPaginationProps = DensityProp & {
    * Whether to show a skeleton pagination.
    */
   skeleton?: SkeletonInjected;
+  /**
+   * The display mode of the pagination
+   * @default select
+   */
+  mode?: OnyxPaginationModes;
 };
+
+export type OnyxPaginationModes = "select" | "inline";
```

Comment:
Imo we should talk with UX about aligning the property values with Figma. I'd prefer using select and inline instead of modern and classic so we name the types after what they are, not how they might be used. Also thats less opinionated about the usage

<img width="515" height="420" alt="Image" src="https://github.com/user-attachments/assets/cd876ce8-5a6c-4e92-a4a4-672168cded17" />